eng/pipelines/maintenance-packages-pr.yml

```diff
@@ -49,6 +49,7 @@ stages:
       publishAssetsImmediately: true
       enablePublishTestResults: true
       testResultsFormat: vstest
+      enableSourceBuild: true
```

Comment:
I hope you don't mind @ericstj that I pushed changes to enable SB legs in PR/CI in response to https://github.com/dotnet/maintenance-packages/pull/140#issuecomment-2427871959.

Reply:
Thanks, I thought I did this, but maybe didn't push the changes to the fork.